src/niscope/metadata/functions.py

```diff
@@ -1346,6 +1346,142 @@
         'python_name': 'get_equalization_filter_coefficients',
         'returns': 'ViStatus'
     },
+    'FancyGetExtCalLastDateAndTime': {
+        'codegen_method': 'python-only',
+        'documentation': {
+            'description': 'Returns the date and time of the last external calibration performed.'
+        },
+        'method_templates': [
+            {
+                'documentation_filename': 'default_method',
+                'method_python_name_suffix': '',
+                'session_filename': 'fancy_get_cal_last_date'
+            }
+        ],
+        'parameters': [
+            {
+                'direction': 'in',
+                'documentation': {
+                    'description': 'Identifies a particular instrument session. You obtain the **vi** parameter from niScope_init.'
+                },
+                'name': 'vi',
+                'type': 'ViSession'
+            },
+            {
+                'direction': 'out',
+                'documentation': {
+                    'description': 'Indicates the **date** of the last calibration. A hightime.datetime object is returned, but only contains resolution to the day.'
+                },
+                'name': 'lastCalDatetime',
+                'python_type': 'hightime.timedelta, datetime.timedelta, or float',
```

Comment:
Remove this key too, when you remove it for `FancyFetch`.

Applies to other fancy methods moved from addon file.